docs/guides/steps-from-apps.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: default
+title: "Workflow Steps"
+lang: en
+---
+
+# Workflow Steps from Apps
+
+Workflow Steps from apps allow your app to create and process custom workflow steps that users can add using [Workflow Builder](https://api.slack.com/workflows).
```

Comment:
The structure is not exactly the same but the contents here are compatible with Bolt for JS/Python.